components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/gen/java/org/wso2/carbon/apimgt/rest/api/admin/dto/SubscriptionThrottlePolicyListDTO.java

```diff
@@ -0,0 +1,105 @@
+package org.wso2.carbon.apimgt.rest.api.admin.dto;
```

Comment:
Fixed  in https://github.com/wso2/carbon-apimgt/commit/8bf295d69103fd9391adef96f70d05a766cb4879.